lib/irb.rb

```diff
@@ -433,6 +433,9 @@ def initialize(workspace = nil, input_method = nil)
     def run(conf = IRB.conf)
       conf[:IRB_RC].call(context) if conf[:IRB_RC]
       conf[:MAIN_CONTEXT] = context
+      conf[:COMMAND_ALIASES].each do |alias_name, cmd_name|
```

Comment:
Can we do this in `Irb#initialize` instead? That way we won't need to duplicate the same logic in tests. 
Also, since we've stored the aliases in `Context`'s `@command_aliases`, can we expose it as reader and use it as the only source of truth?

Reply:
> Can we do this in Irb#initialize instead? That way we won't need to duplicate the same logic in tests.

What about the `conf = IRB.conf` in the argument? It seems weird to handle `IRB.conf` in `#initialize` when you could override it in the argument of `#run`.

> Also, since we've stored the aliases in Context's @command_aliases

What do you mean? We only have `@symbol_aliases` to achieve your "Hash look up" idea. If you only use `@symbol_aliases` here, it will not alias non-symbol aliases.

I can change it to `@command_aliases` and fix other stuff though.

Reply:
addressed it in https://github.com/ruby/irb/pull/426/commits/02e9369c059b4c45e829d712bb26610f1a3453b8